### PR TITLE
Fish factory replace conveyor belts

### PR DESCRIPTION
**Pull request by mefriis**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Feature: Replace conveyor belts and sliders

* **What is the current behavior?** (You can also link to an open issue here)
#354 

* **What is the new behavior?** (if this is a feature change)
Adds new conveyor belts from model
Adds scrolling texture movements on conveyors
Adds new sliders 
Visual updates

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

File: Assets/FishFactory/Scripts/Common/ConveyorController.cs [hunk 1]
```diff
@@ -6,10 +6,12 @@ public class ConveyorController : MonoBehaviour
 {
     private enum Direction
     {
-        Forward,
-        Backward,
-        Right,
-        Left,
+        Forward_BlueI,
```
Comment (jonp-h) [inline Assets/FishFactory/Scripts/Common/ConveyorController.cs:9]: What is BlueI, D, etc?

**Review by jonp-h — COMMENTED** [review]
Unclear enum? Else good to merge

File: Assets/FishFactory/Scripts/Common/ConveyorController.cs (hunk 1 above)
Comment (mefriis) [reply Assets/FishFactory/Scripts/Common/ConveyorController.cs:9]: Names refering to the color used by unity to indicate movement axis: Forward and Backward on Blue axis, Left and Right on Red axis and Up and Down on Green axis. D and I refers to whether it is an increase or decrease in value along the given axis. If these variable names is not suitable, I can rename back to original, or add a comment in the code explaining the names?

**Review by krisgjol — APPROVED** [review]
Dont care too much about the enums, if we want to clean it up, we can do it later

File: Assets/FishFactory/Scripts/Common/ConveyorController.cs (hunk 1 above)
Comment (jonp-h) [reply Assets/FishFactory/Scripts/Common/ConveyorController.cs:9]: Should be good

**Pull request merged**